test/helpers/test-stories.js

```diff
@@ -0,0 +1,130 @@
+import {
+  ignoreWindowOnError,
+  isResizeObserverLoopErrorMessage,
+} from '@lit-labs/virtualizer/support/resize-observer-errors.js';
+import { elementUpdated, expect, fixture } from '@open-wc/testing';
+import { setViewport } from '@web/test-runner-commands';
+import { addTranslations } from '../../src/lib/i18n/i18n.js';
+import * as en from '../../src/translations/translations.en.js';
+
+/**
+ * @typedef {import('./test-stories.types.js').RawStoriesModule} RawStoriesModule
+ * @typedef {import('./test-stories.types.js').AnnotatedStoryFnClever} AnnotatedStoryFnClever
+ * @typedef {import('./test-stories.types.js').ModuleEntryWithStoryFunction} ModuleEntryWithStoryFunction
+ * @typedef {import('./test-stories.types.js').ArrayOfStoryFunctions} ArrayOfStoryFunctions
+ * @typedef {import('@storybook/web-components').WebComponentsRenderer} WebComponentsRenderer
+ * @typedef {import('@storybook/web-components').StoryContext<WebComponentsRenderer>} StoryContext
+ */
+
+const viewports = {
+  desktop: {
+    width: 1200,
+    height: 640,
+  },
+  mobile: {
+    width: 360,
+    height: 640,
+  },
+};
+
+/** @type {StoryContext} */
+// @ts-ignore we don't need to provide all options because the story won't be displayed in storybook
```

Comment:
question: couldn't we use a `Partial<StoryContext>`?

Reply:
No because `storyFunction` expects a full `StoryContext` (line 103 & 116).

This is something I'll try to improve when I type makeStory properly.